lib/elixir/lib/system.ex

```diff
@@ -367,7 +367,7 @@ defmodule System do
   `varname` as a binary, or `nil` if the environment
```

Comment:
"`varname` as a binary" could also be "`varname` as a string", not sure what are rules around calling thins binaries/strings